modules/home/assets/js/components/promotions/sidebar-banner.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Paper } from '@elementor/ui';
+import Link from '@elementor/ui/Link';
+
+const SidebarBanner = ( { image, link } ) => {
+	return (
+		<Paper elevation={ 0 } sx={ { overflow: 'hidden' } }>
+			<Link
+				target="_blank"
+				href={ link }
+				sx={
+					{
+						display: 'block',
+						width: '100%',
+						height: '100%',
+						boxShadow: 'none',
+						outline: 'none',
```

Comment:
@mykytamurzin Would it be possible to keep this outline for accessibility reasons? I don't think it is good practice to remove the focus outline on a link tag.